api/src/pcapi/routes/pro/offers.py

```diff
@@ -104,10 +104,7 @@ def delete_draft_offers(body: offers_serialize.DeleteOfferRequestBody) -> None:
     api=blueprint.pro_private_schema,
 )
 def post_offer(body: offers_serialize.PostOfferBodyModel) -> offers_serialize.GetIndividualOfferResponseModel:
-    venue_id = human_ids.dehumanize(body.venue_humanized_id)
-    if not venue_id:
-        raise api_errors.ApiErrors(errors={"venueId": ["L'id n'est pas au bon format"]}, status_code=404)
-    venue: offerers_models.Venue | None = offerers_models.Venue.query.get(venue_id)
+    venue: offerers_models.Venue | None = offerers_models.Venue.query.get_or_404(body.venue_id)
```

Comment:
Puisqu'on utilise désormais `get_or_404()`, on peut désormais supprimer le bloc `if not venue` en dessous.

C'est dans le commit "dehumanize POST /offers".